docs/settings.rst

```diff
@@ -28,6 +28,7 @@ Some of Simple JWT's behavior can be customized through settings variables in
       'ISSUER': None,
 
       'AUTH_HEADER_TYPES': ('Bearer',),
+      'AUTH_USES_X_ACCESS_TOKEN': False,
```

Comment:
It might be better if we just used any header the developer provides. So it would be `AUTH_HEADER_NAME`